paf-mvp-benchmark/Docker/Collector/README.md

```diff
@@ -0,0 +1,12 @@
+## Create a virtual network
+docker network create observability-network
+
+## Launch OTLP collector
+Go to paf-mv-benchmark/Collector and run:\
```

Comment:
typo in `mvp`

Reply:
Path should be updated to `paf-mvp-benchmark/Docker/Collector`

Reply:
Done